content/blog/2021-04-30-april-21-community-gems.md

```diff
@@ -0,0 +1,235 @@
+$ dvc exp run --run-all --jobs 2
+```
+
+Then you can compare the results with `dvc exp show`.
+
+```
+┏━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
+┃ Experiment    ┃ avg_prec ┃ roc_auc ┃ train.n_est┃ train.min_split ┃
+┡━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
+│ workspace     │  0.56191 │ 0.93345 │ 50         │ 2               │
+│ master        │  0.55259 │ 0.91536 │ 50         │ 2               │
+│ ├── exp-bfe64 │  0.57833 │ 0.95555 │ 50         │ 8               │
+│ ├── exp-b8082 │  0.59806 │ 0.95287 │ 50         │ 64              │
+│ ├── exp-c7250 │  0.58876 │ 0.94524 │ 100        │ 2               │
+│ ├── exp-b9cd4 │  0.57953 │ 0.95732 │ 100        │ 8               │
+│ ├── exp-98a96 │  0.60405 │  0.9608 │ 100        │ 64              │
+│ └── exp-ad5b1 │  0.56191 │ 0.93345 │ 50         │ 2               │
+└───────────────┴──────────┴─────────┴────────────┴─────────────────┘
+```
+
+We are working on developing experiments to have features or documented patterns
+explicitly for grid search support, so definitely
+[share any feedback](https://github.com/iterative/dvc/issues/4283) to help drive
+the future direction of that!
+
+### [When importing/getting data from a repo, how do I provide credentials to the source repo remote storage without saving it into that Git repo?](https://discord.com/channels/485586884165107732/563406153334128681/830021022337073185)
+
+There's a bit of context behind this question that might give it more meaning.
+Here's the background information given by @EdAb in Discord:
+
+---
+
+I set up a private GitHub repo to be a data registry and I have set up a private
+Azure remote where I have pushed some datasets.
+
+I am now trying to read those datasets from another repository
+("my-project-repo"), using `dvc get` (e.g.
+`dvc get git@github.com:data-registry-repo.git path/data.csv`) but I get this
+error:
+
+```bash
+ERROR: failed to get 'path/data.csv' from 'git@github.com:data-registry-repo.git' - Authentication to Azure Blob Storage via default credentials (https://azuresdkdocs.blob.core.windows.net/$web/python/azure-identity/1.4.0/azure.identity.html#azure.identity.DefaultAzureCredential) failed.
+Learn more about configuration settings at <https://man.dvc.org/remote/modify>: unable to connect to account for Must provide either a connection_string or account_name with credentials!!
+```
+
+---
+
+Generally, there are two ways solve this issue:
+
+- [ENV vars](https://docs.microsoft.com/en-us/python/api/azure-identity/azure.identity.environmentcredential?view=azure-python)
+- Setup some options using the `--global` or `--system` flags to update the DVC
+  config
+
+If you're going to update the DVC config to include your cloud credentials, use
+the `dvc remote modify` command. Here's an example of how you can do that with
+Azure using the `--global` flag.
+
+```dvc
+$ dvc remote modify --global myremote connection_string 'mysecret'
```

Comment:
we should clarify where does `myremote` name come from in this case

also, most likely running this command like this will end up in a error (remote not found) - first they should initialize the remote in that config file with `dvc remote add` and remove `url` (to rely on the one that comes from the repo that is being imported - this unfortunate DVC limitation at this point).

I would recommend to try to run this scenario to actually hit all the problems.